scripts/summarize_celestia_sync_runs_test.py

```
@@ -380,13 +417,84 @@ def test_treedb_maintenance_summary_selects_app_instance_by_wal_dir(self) -> Non
+                encoding="utf-8",
+            )
+            out = root / "out"
+
+            result = subprocess.run(
+                [str(SCRIPT), "--out-dir", str(out), str(run)],
+                cwd=ROOT,
+                text=True,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+                check=False,
+            )
```

Ruff (0.15.20)

[warning] Prefer `capture_output` over sending `stdout` and `stderr` to `PIPE`

Replace with `capture_output` keyword argument

(UP022)